### PR TITLE
Add camel examples

### PR DESCRIPTION
**Pull request by ThePrez**
WIP

### CODE REVIEW

**Comment by ThePrez** [comment]
We need to wait for Camel release 3.5.0 before landing this, since it has dependency on MSGQ support which is new. 
Otherwise, PR is ready for early review/feedback. Will undraft when camel 3.5.0 hits

File: camel/dtaq_to_kafka/src/main/java/com/ibm/jesseg/MainApp.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        final String dtaqUri = conf.getDtaQUri(); //something like -> jt400://username:password@localhost/qsys.lib/mylib.lib/myq.DTAQ?keyed=false&format=binary&guiAvailable=false
+        final String smtpUri = conf.getKafkaUri(); //something like -> kafka:mytopic?brokers=mybroker:9092
+        context.addRoutes(new RouteBuilder() {
+            @Override
+            public void configure() {
+                from(dtaqUri)
+                .wireTap("log:msgq_to_email?showAll=true&level=INFO") // This is just for debugging data flowing through the route
+                .to(smtpUri); 
+            }
+        });
```
Comment (kadler) [inline camel/dtaq_to_kafka/src/main/java/com/ibm/jesseg/MainApp.java:32]: Might want to give more detail here about what's going on. This is essentially the crux of the example and it's seemingly smooshed in with getting some URIs.

Comment (ThePrez) [reply camel/dtaq_to_kafka/src/main/java/com/ibm/jesseg/MainApp.java:32]: Good point. I'll write up some info here. I had initially thought about walking through the snippet in the main README but ended up forgetting completely.

**Comment by ThePrez** [comment]
Welp, I accidentally deleted my fork. Doing the lazy thing and merging this in, even though v3.5.0 of Camel isn't out yet.

**Pull request merged**